beamer/models/request.py

```diff
@@ -31,24 +31,35 @@ def __init__(
         self.filler: Optional[ChecksumAddress] = None
         self.fill_tx: Optional[HexBytes] = None
         self.fill_id: Optional[int] = None
+        self.l1_resolution_filler: Optional[ChecksumAddress] = None
 
     pending = State("Pending", initial=True)
     filled = State("Filled")
     claimed = State("Claimed")
+    l1_resolved = State("L1Resolved")
```

Comment:
Could you please also update the state machine diagram in the docs to reflect this new state?

Reply:
Sure